### ISSUE TITLE
Feature request: add device models

### ISSUE THREAD

**Issue by DrBlokmeister** [arjenbos/ha_alpha_innotec#23]
It would be nice to have device models display in the integration page. I think currently the most accurate model would be the module type, so sensor, sense_control or valve.

**Comment by arjenbos** [comment]
Are you referring to the Home Assistant [device models](https://developers.home-assistant.io/docs/device_registry_index/)?

> The model of the device.

Maybe the model of your heat pump can be added. A device can consist out of multiple sensors. So, adding a model for a device with multiple sensors will be a problem.

**Comment by ChrisMisker** [comment]
Hi @DrBlokmeister , are you referring to these?
<img width="1184" alt="image" src="https://github.com/arjenbos/ha_alpha_innotec/assets/37826263/9b7413e9-2060-4639-9633-cc2005357231">

@arjenbos Is it possible to change the model here? Perhaps "thermostat" and "valve" and such should be a good name for the model? The Gateway knows what type of sensor it is, or it couldn't show the images and such...
<img width="874" alt="image" src="https://github.com/arjenbos/ha_alpha_innotec/assets/37826263/95a24180-77da-4345-9d04-c41febacbbe9">

**Comment by DrBlokmeister** [comment]
Yes! These exactly. I would indeed make it Sensor/Thermostat (or Sense/Control)/Valve.
